number_inputs.py

```diff
@@ -0,0 +1,21 @@
+"""
+number_inputs asks for a number 5 times and
+outputs the first, last, smallest, largest and average of the numbers.
+"""
+__author__ = 'Gareth'
+
+
+def main():
+    numbers = []
+    NUM_OF_NUMBERS = 5
+    for i in range(0, NUM_OF_NUMBERS):
+        numbers.append(int(input(" please enter number {} ".format(i + 1))))
+    print("The first number is {}".format(numbers[0]))
```

Comment:
I would need to create a function for each print, then either call each function or call another function to call each function, therefore, I don't think putting them into function(s) would be beneficial